scripts/skills/perks/perk_rf_kingfisher.nut

```diff
@@ -21,7 +22,7 @@ this.perk_rf_kingfisher <- ::inherit("scripts/skills/skill", {
 	{
 		if (this.m.IsForceEnabled) return true;
 
-		return this.getContainer().hasSkill("actives.throw_net");
+		return this.getContainer().hasSkill("actives.throw_net") != null; // Ignores hidden skills so while this IsSpent, it will be false
```

Comment:
This is not necessary and misleading.
The Vanilla `hasSkill` function always returns either `true` or `false`.

Reply:
Ah yes the `!= null` is there by mistake. Will fix.